kernel/include/paging.h

```diff
@@ -12,29 +39,30 @@
 typedef uint64_t PageFlags;
 typedef uint64_t PageEntry;
 
-#define PAGETABLE_SIZE 0x100000
 
-//temporary solution before memory allocation is done
-extern uint64_t page_zone_stack[PAGETABLE_SIZE];
-extern uint64_t* page_zone_pointer;
+#define PAGETABLES_SIZE 0xFA000
+
+//this is where all pagetables are stored, we don't want to run out.
+extern uint64_t page_tables[PAGETABLES_SIZE];
```

Comment:
these globals shouldn't be in the header because they don't need to be used outside of the source file